concent_api/utils/logging.py

```diff
@@ -153,21 +167,30 @@ def log_subtask_updated(
 
 
 def log_stored_message_added_to_subtask(
-    task_id:              str,
-    subtask_id:           str,
-    state:                str,
+    logger,
+    task_id: str,
+    subtask_id: str,
+    state: str,
     stored_message:  Message,
+    provider_public_key: bytes,
+    requestor_public_key: bytes
 ):
-    logger.info('A stored message has beed added to subtask -- STATE: {} TASK_ID: {} SUBTASK_ID: {} STORED_MESSAGE_TYPE: {} TYPE: {}'.format(
+    logger.info('A stored message has beed added to subtask -- STATE: {} TASK_ID: {} SUBTASK_ID: {} STORED_MESSAGE_TYPE: {} TYPE: {} PROVIDER PUBLIC KEY: {} REQUESTOR PUBLIC KEY: {}'.format(
         state,
         task_id,
         subtask_id,
         stored_message.__name__,
         stored_message.TYPE,
+        provider_public_key,
+        requestor_public_key
     ))
 
 
-def log_changes_in_subtask_states(client_public_key: bytes, count: int):
+def log_changes_in_subtask_states(
+    logger,
+    client_public_key: bytes,
+    count: int
+):
     assert isinstance(count, int)
     logger.info('{} {} state -- CLIENT PUBLIC KEY: {}'.format(
```

Comment:
If you are editing this file you can also change these strings format to f-string. It will make this code looks better

Reply:
done